include/swift/Sema/ConstraintSystem.h

```diff
@@ -356,6 +356,10 @@ class TypeVariableType::Implementation {
   /// Determine whether this type variable represents a closure result type.
   bool isClosureResultType() const;
 
+  /// Determine whether this type variable represetns
```

Comment:
Typo: represetns -> represents :)

Reply:
Let's see if this run of macOS succeeds if not - I'm going to fix it here...

Reply:
Fixed, thank you!